src/lib.rs

```diff
@@ -1546,7 +1420,7 @@ impl error::Error for SetLoggerError {}
 ///
 /// [`from_str`]: https://doc.rust-lang.org/std/str/trait.FromStr.html#tymethod.from_str
 #[allow(missing_copy_implementations)]
-#[derive(Debug, PartialEq)]
+#[derive(Debug, PartialEq, Eq)]
```

Comment:
Functional/API change, but personally I'm ok with it.

Reply:
Clippy suggested it and I thought it made sense.